package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-flash",
-  "version": "1.9.0",
+  "version": "1.10.0",
```

Comment:
Can we not include a version bump here?  Since we drop support for older versions of ember and require newer versions of node, I would consider this a major version.  I can cut that as a separate PR